packages/components/src/Heading/Heading.tsx

```diff
@@ -14,6 +14,7 @@ type Props = {
    */
   as?: HeadingElement;
   children: TypographyProps<HeadingElement>["children"];
+  className: TypographyProps<HeadingElement>["className"];
```

Comment:
should this be optional?

Reply:
yes, but at the typography level (which is also missing), this will inherit that property. I'll add that, nice catch 👍